cmd/web-hooks/internal/handler/handler.go

```diff
@@ -522,6 +541,36 @@ func (wh *WebhookHandler) validateCAPTenant(w http.ResponseWriter, admissionRevi
 	return validAdmissionReviewObj()
 }
 
+func (wh *WebhookHandler) validateCAPTenantOutput(w http.ResponseWriter, admissionReview *admissionv1.AdmissionReview) validateResource {
+	ctoutObjNew := ResponseCtout{}
+
+	if admissionReview.Request.Operation == admissionv1.Create {
```

Comment:
could you change this to both create and update?

Reply:
Check now being done for both create and update